packages/notte-sdk/src/notte_sdk/types.py

```
@@ -29,6 +29,13 @@
 DEFAULT_MAX_NB_STEPS = 20
 
 
+class PlaywrightProxySettings(TypedDict, total=False):
```

Ruff (0.8.2)

Redefinition of unused `PlaywrightProxySettings` from line 17

(F811)